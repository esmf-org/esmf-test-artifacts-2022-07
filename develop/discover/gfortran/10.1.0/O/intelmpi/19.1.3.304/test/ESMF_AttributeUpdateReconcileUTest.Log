build time -- 2022-07-06 01:09:47.499868
20220706 012352.375 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220706 012352.375 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220706 012352.375 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220706 012352.375 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220706 012352.375 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220706 012352.375 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220706 012352.375 INFO             PET0 Running with ESMF Version   : v8.4.0b03
20220706 012352.375 INFO             PET0 ESMF library build date/time: "Jul  6 2022" "01:08:34"
20220706 012352.375 INFO             PET0 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/gfortran_10.1.0_intelmpi_O_develop
20220706 012352.375 INFO             PET0 ESMF_COMM                   : intelmpi
20220706 012352.376 INFO             PET0 ESMF_MOAB                   : enabled
20220706 012352.376 INFO             PET0 ESMF_LAPACK                 : enabled
20220706 012352.376 INFO             PET0 ESMF_NETCDF                 : disabled
20220706 012352.376 INFO             PET0 ESMF_PNETCDF                : disabled
20220706 012352.376 INFO             PET0 ESMF_PIO                    : disabled
20220706 012352.376 INFO             PET0 ESMF_YAMLCPP                : enabled
20220706 012352.376 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
20220706 012352.376 INFO             PET0  NUMBER_OF_PROCESSORS           2
20220706 012352.378 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
20220706 012352.378 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
20220706 012352.378 INFO             PET0  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
20220706 012352.378 INFO             PET0 Finalizing ESMF
20220706 012352.376 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220706 012352.376 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220706 012352.376 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220706 012352.376 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220706 012352.376 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220706 012352.376 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220706 012352.376 INFO             PET1 Running with ESMF Version   : v8.4.0b03
20220706 012352.376 INFO             PET1 ESMF library build date/time: "Jul  6 2022" "01:08:34"
20220706 012352.376 INFO             PET1 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/gfortran_10.1.0_intelmpi_O_develop
20220706 012352.376 INFO             PET1 ESMF_COMM                   : intelmpi
20220706 012352.376 INFO             PET1 ESMF_MOAB                   : enabled
20220706 012352.376 INFO             PET1 ESMF_LAPACK                 : enabled
20220706 012352.376 INFO             PET1 ESMF_NETCDF                 : disabled
20220706 012352.376 INFO             PET1 ESMF_PNETCDF                : disabled
20220706 012352.376 INFO             PET1 ESMF_PIO                    : disabled
20220706 012352.376 INFO             PET1 ESMF_YAMLCPP                : enabled
20220706 012352.376 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
20220706 012352.376 INFO             PET1  NUMBER_OF_PROCESSORS           2
20220706 012352.378 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
20220706 012352.379 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
20220706 012352.379 INFO             PET1  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
20220706 012352.379 INFO             PET1 Finalizing ESMF
