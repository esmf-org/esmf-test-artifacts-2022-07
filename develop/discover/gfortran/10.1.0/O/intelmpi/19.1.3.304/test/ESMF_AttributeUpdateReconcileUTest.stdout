build time -- 2022-06-20 01:09:39.998719
srun: cluster configuration lacks support for cpu binding
  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
  NUMBER_OF_PROCESSORS           2
  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
  NUMBER_OF_PROCESSORS           2
  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
  PET           0  Test Elapsed Time    9.8779999999999983       msec.
  PET           1  Test Elapsed Time    13.356000000000007       msec.
